src/main/java/se/liu/ida/hefquin/engine/data/impl/VocabularyMappingImpl.java

```diff
@@ -0,0 +1,23 @@
+package se.liu.ida.hefquin.engine.data.impl;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.riot.RDFDataMgr;
+
+import se.liu.ida.hefquin.engine.data.VocabularyMapping;
+
+public class VocabularyMappingImpl implements VocabularyMapping{
+	
+	Model vocabularyMapping;
```

Comment:
```suggestion
	protected final Model vocabularyMapping;
```

Reply:
Note that the previous comment is a change suggestion which you can accept and apply directly here in the PR. Doing so creates a new commit for your branch. Therefore, after accepting such suggestions, before you start implementing more changes on your computer, you need to do a `git pull` in your local branch in order to receive these new commits into your local branch.